packages/client/hmi-client/src/components/workflow/util.ts

```diff
@@ -5,12 +5,26 @@ export const enum RunType {
 	Ciemss = 'ciemss'
 }
 
-export const getTimespan = (inputTimespan: TimeSpan, dataset?: CsvAsset) => {
+export type DatasetType = {
+	data: { x: number; y: number }[];
+	label: string;
+	fill: boolean;
+	borderColor?: string;
+	borderWidth?: number;
+	borderDash?: number[];
+};
```

Comment:
The naming does not reflect the purpose of this data structure. 

Probably more appropriate as `DataseriesConfig` or something?
Also consider moving the definition over to types/workflow.ts or types/SimulateConfig.ts

Reply:
name updated and moved to `SimulateConfig.ts`